go.mod

```diff
@@ -5,13 +5,13 @@ go 1.17
 require (
 	cloud.google.com/go/bigtable v1.13.0
 	github.com/google/go-cmp v0.5.7
-	github.com/jhump/protoreflect v1.11.0
+	github.com/jhump/protoreflect v1.11.0 // Third-party dependency; proceed with caution
 	golang.org/x/oauth2 v0.0.0-20211104180415-d3ed0bb246c8
 	golang.org/x/sys v0.0.0-20220209214540-3681064d5158
 	google.golang.org/api v0.67.0
 	google.golang.org/grpc v1.44.0
 	google.golang.org/protobuf v1.27.1
-	gopkg.in/yaml.v2 v2.2.3
+	gopkg.in/yaml.v2 v2.2.3 // Third-party dependency; proceed with caution
```

Comment:
I wonder if this line will just get overwritten by renovate bot updates?

Reply:
Maybe! Let's cross that bridge when we come to it.